packages/data/src/world/mm/snowhead_temple.yml

```diff
@@ -31,9 +31,9 @@
     RUPEES: "true"
   exits:
     "Snowhead Temple Main": "true"
-    "Snowhead Temple Bridge Back": "goron_fast_roll || can_hookshot || short_hook_anywhere"
+    "Snowhead Temple Bridge Back": "goron_fast_roll || can_hookshot || short_hook_anywhere || has_hover_boots"
   locations:
-    "Snowhead Temple Bridge Room": "soul_freezard && can_hookshot_short"
+    "Snowhead Temple Bridge Room": "soul_freezard && (can_hookshot_short || (has_hover_boots && has_weapon))"
```

Comment:
For now, there doesnt need to be an item check with hover boots since you're guaranteed goron or zora within the dungeon. But for the future, it would also include explosives, goron, or zora.

Reply:
Will remove `has_weapon` since that's not only redundant but it adds an unnecessary sword requirement.